### PR TITLE
DRIVERS-2795 Apply automated linting and formatting fixes

### PR DESCRIPTION
**Pull request by blink1073**
Applies the pre-commit fixes using https://github.com/mongodb/mongo-python-driver/blob/master/.pre-commit-config.yaml.

Follow up PR will add the `.git-blame-ignore-revs` file, and the associated linting configuration.

### CODE REVIEW

**Comment by blink1073** [comment]
@Jibola the only manual change I made was in [atlasclient/__init__.py](https://github.com/mongodb-labs/drivers-atlas-testing/pull/187/files#diff-8d81ed7f835988df5946d595613d5d871c64437997f8409a0703a7fcd1cc7206), to prevent these imports from being removed.

**Review by Jibola — APPROVED** [review]

**Comment by blink1073** [comment]
I'll wait until Monday to merge this, in case things go wrong...

**Pull request merged**
